src/AzureStack/GalleryAdmin/Gallery.Admin/Microsoft.AzureStack.Management.Gallery.Admin.csproj

```diff
@@ -1,4 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
nit: no useful changes here

Reply:
Leaving so that all the Admin csproj files have the exact same structure.